From 66a6f7f50ceaccf47dcab12a0d26cb1eab861cc4 Mon Sep 17 00:00:00 2001
From: Thomas Orozco <thomas@orozco.fr>
Date: Mon, 4 Apr 2016 21:31:01 +0200
Subject: [PATCH] Add container_id as tag

Dereferenced aliases can be nice, but a predictable container ID is nice
too.
---
 storage/influxdb/influxdb.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/storage/influxdb/influxdb.go b/storage/influxdb/influxdb.go
index 82739b5..4aeb89d 100644
--- a/storage/influxdb/influxdb.go
+++ b/storage/influxdb/influxdb.go
@@ -101,6 +101,7 @@ const (
 const (
 	tagMachineName   string = "machine"
 	tagContainerName string = "container_name"
+	tagContainerId   string = "container_id"
 )
 
 func (self *influxdbStorage) containerFilesystemStatsToPoints(
@@ -158,6 +159,7 @@ func (self *influxdbStorage) tagPoints(ref info.ContainerReference, stats *info.
 	commonTags := map[string]string{
 		tagMachineName:   self.machineName,
 		tagContainerName: containerName,
+		tagContainerId:   ref.Name,
 	}
 	for i := 0; i < len(points); i++ {
 		// merge with existing tags if any
-- 
2.7.4

